hash/sms.xml

```diff
@@ -8270,4 +8270,16 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="3dadp">
+		<description>3-D Adaptor</description>
+		<year>1987</year>
+		<publisher>Sega</publisher>
+		<part name="cart" interface="sms_card">
+			<feature name="slot" value="3dadp" />
+			<dataarea name="rom" size="1">
+				<!-- this card is just an adapter -->
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Is this really necessary?  Shouldn’t you be able to instantiate the device using the slot option directly rather than makign it an internal option and making a fake software list item like this?

Reply:
I tested using the slot option directly (using -mycard) to load the card adapter after changing it to not be an internal option and it works, but unfortunately MAME also allows to load another card media at same time (using -card), what is not possible in real hardware, where only one card can fit in the card slot. Is there a way to disable the load of software in the card slot when using it directly? If yes, maybe would be better to rename "-mycard" to "-cardslot" or something similar due to be used not only for Sega MyCards anymore.